pkg/websocketclient/websocketclient.go

```diff
@@ -284,5 +373,59 @@ func (c *WebsocketClient) ensureConnection() error {
 		}
 	}
 
+	go func() {
+		err := c.listen()
+		if err != nil {
+			log.Printf("Error calling listen: %s\n", err.Error())
```

Comment:
I could see an application writer finding this logline kind of confusing/contextless. Makes sense here, but not out in the world in somebody's poker card game